webrtc.html

```diff
@@ -4587,8 +4587,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             class="methods">
               <dt><dfn><code>getAlgorithm</code></dfn></dt>
               <dd>
-                <p>Returns the value of <var>keygenAlgorithm</var> passed in
-                the call to <code>generateCertificate()</code>.</p>
+                <p>Returns an AlgorithmIdentifier corresponding to the one passed in the call to
+                <code>generateCertificate()</code>, meaning an object that would produce the same
+                result by the normalization process as the original <var>keygenAlgorithm</var>
```

Comment:
Why not just "the result of the normalization process", as opposed to "an object that would produce the same result", so that the return value is 100% predictable? Also, this could link to https://www.w3.org/TR/WebCryptoAPI/#dfn-normalize-an-algorithm